### DISCUSSION TITLE
blog/2024/05/08/coastline-to-border-proportions/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#97] [Blog comments]
# blog/2024/05/08/coastline-to-border-proportions/

Measuring coastlines is hard and causes fractal paradoxes, but we can use R and {sf} to try!

https://www.andrewheiss.com/blog/2024/05/08/coastline-to-border-proportions/?utm_campaign=Data_Elixir&utm_source=Data_Elixir_485

**Comment by talegari** [comment]
TLDR, `st_buffer` |> `st_difference` |> `divide_by(2)` is a simple and elegant idea! Also a ton of new learning about `sf` and friends! Great blog Andrew.
